content/_releases/22/16.md

```diff
@@ -18,11 +18,11 @@ If you are one of power users of our platform you might have noticed a pop-up wa
 with a message **Not enough quota to start task** once or twice. In reality this
```

Comment:
These messages were not related to your RAM quota limit. Instead these messages are triggered by real-time flows with more than 25 steps. Because the platform initiates each step sequentially, these errors are triggered when all flow steps cannot be initiated within the allowed time.  

We have resolved this issue by ensuring that all steps (Kubernetes pods) are initiated in parallel, thus significantly shortening the time required to start the flow.

Reply:
Added as suggested